### ISSUE TITLE
[Test failure] <TC_AUTHOR_86> Inconsistent Behavior with 'Hide Discussion Tab' Toggle and Discussion Sidebar Accessibility

### ISSUE THREAD

**Issue by HeliaEnayati** [openedx/wg-build-test-release#417]
### Release

sumac

### Expected behavior

When the 'Hide Discussion Tab' toggle is enabled and saved, the interface should reflect this change without requiring a refresh of the Page and Resources page. Clarification is needed on the expected behavior regarding the sidebar's accessibility when the discussion tab is hidden.

### Actual behavior

1. After enabling the "Hide Discussion Tab" toggle and clicking "Save," you are redirected back to the Page and Resources page. Up to this point, everything is functioning and the discussion tab will be hidden as expected.
 
3. The "Enabled" label remains on the discussion card until the page is refreshed

4. In the course outline, the unit discussion configuration settings relating to the discussion sidebar can still be checked or unchecked, but this does not result in any changes. I understand that enabling the 'Hide Discussion Tab' toggle prevents learners from posting in the discussion forums. However, I think enabling the 'Hide Discussion Tab' toggle should either disable the ability to make changes to the unit discussion configuration or allow learners to see the sidebar but not post in it. This inconsistency in labels may confuse the instructor.

### Steps to reproduce

1. Log in to the studio
2. Navigate to the Page and Resources section.
3. Click on the configure icon and enable the "Hide Discussion Tab" toggle.
4. Click "Save" and observe the behavior on the Page and Resources page regarding the "Enabled" label.
5. Refresh the page and note any changes in the label's visibility.
6. Check the course outline and unit cards to see if the discussion status reflects the toggle changes.

**Comment by mariajgrimaldi** [comment]
The hide discussion tab doesn't disable the discussions service altogether, it seems to be that just hides the tab for learners, as the confirmation message says:

> The discussion tab will no longer be visible to learners in the LMS. Additionally, posting to the discussion forums will be disabled. Are you sure you want to proceed?

Hiding the discussion tab didn't remove the `enable` chip even after reloading for me. I had to do some maneuvers in the Django admin to make it disappear. Your report is pretty detailed and thorough, so thank you for that. But can you confirm again that the `enable` chip disappeared after reloading? Cause I can't reproduce it.

I believe the configuration from Pages & Resources only affects course-wide settings. It hides the course discussion tab in the LMS and configures the discussion provider, but it doesn't disable the discussion service per se, so [in-line discussions are still available](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3470655498/Discussions+upgrade+Sidebar+and+new+topic+structure#How-to-enable/disable-discussion-on-course-units). 

I can't find anything in the docs explicitly saying how to disable discussions altogether besides setting `FEATURES["ENABLE_DISCUSSION_SERVICE"] = False` in the sandbox environment.

**Comment by mariajgrimaldi** [comment]
@crathbun428: could you give us a hand with? _Clarification is needed on the expected behavior regarding the sidebar's accessibility when the discussion tab is hidden._

I can't find any docs about this anywhere, maybe I'm missing something.

**Comment by crathbun428** [comment]
@mariajgrimaldi - your read of how this works sounds right to me: The configuration from Pages & Resources only affects course-wide settings. It hides the course discussion tab in the LMS and configures the discussion provider, but it doesn't disable the discussion service per se, so [in-line discussions are still available](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3470655498/Discussions+upgrade+Sidebar+and+new+topic+structure#How-to-enable/disable-discussion-on-course-units).

**Comment by mariajgrimaldi** [comment]
@crathbun428 thank you! I'm going to close this since it is a non-issue.
